packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/RowLabelWithCheckbox/index.js

```diff
@@ -1,11 +1,16 @@
-import { Row, Checkbox, Text } from '@strapi/parts';
+import styled from 'styled-components';
+import { Row, BaseCheckbox, Text } from '@strapi/parts';
 import upperFirst from 'lodash/upperFirst';
 import PropTypes from 'prop-types';
 import React, { memo } from 'react';
 import { useIntl } from 'react-intl';
 import CollapseLabel from '../CollapseLabel';
 import { firstRowWidth } from '../Permissions/utils/constants';
 
+const CustomCheckbox = styled(BaseCheckbox)`
+  margin-right: ${({ theme }) => theme.spaces[2]};
```

Comment:
Could you please wrap the BaseCheckbox with a Box and use padding instead? 🙏🏻

Reply:
makes sense 👌